plone/app/z3cform/templates/checkbox_input.pt

```diff
@@ -97,7 +97,7 @@
                onselect view/onselect;
              "
       />
-      <label class="form-check-label"
+      <label class="form-check-label ${python:getattr(view, 'label_css_class', False)}"
```

Comment:
If no `label_css_class` is found in `view`, will that render `class="form-check-label False"` then? 🤔

Reply:
it's the same code as here:
https://github.com/plone/plone.app.z3cform/blob/435724666b6134dd7ecf6062cacaf8c3f36de181/plone/app/z3cform/templates/widget.pt#L1
but you are right, i fixed it in both places.